src/main/java/sirius/biz/tycho/kb/KnowledgeBaseController.java

```diff
@@ -67,6 +67,7 @@ public void article(WebContext webContext, String articleId) {
      * @param lang       the language to render the article in
      * @param articleId  the id or code of the article to render
      */
+    @LoginRequired
```

Comment:
This is handled in "resolve" and should not be enforced here, as there might as well be "public" articles.